docs/rules/no-loose-assertions.md

```diff
@@ -0,0 +1,87 @@
+# Forbid the use of assert.equal/assert.ok/assert.notStrictEqual/assert.notOk (no-loose-assertions)
+
+The `assert.equal` assertion method in QUnit uses loose equality comparison. In a project which favors strict equality comparison, it is better to use `assert.strictEqual` for scalar values and either `assert.deepEqual` or `assert.propEqual` for more complex objects.
+
+`assert.ok` and `assert.notOk` pass for any truthy/falsy argument. As [many expressions evaluate to true/false in JavaScript](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) the usage of `assert.ok` is potentially error prone. In general, it should be advisable to always test for exact values in tests which makes tests a lot more solid.
+
+An example when using `assert.ok` can involuntarily go wrong:
+```
+test('test myFunc returns a truthy value' (assert) => {
+  assert.ok(myFunc);
+});
+```
+Here by mistake a developer just passed to `assert.ok` a pointer to `myFunc` instead of explicitly calling it. This test is going pass no matter how `myFunc` changes. Using `assert.strictEqual(myFunc, theReturnValue)` solves the problem as this becomes an explicit check for equality.
+
+The assertions to lint against can be controlled with an array of assertions (default `["equal", "ok", "notOk"]`).
```

Comment:
```suggestion
The assertions to lint against can be controlled with an array of assertions (default `["equal", "notEqual", "ok", "notOk"]`).
```

Note: This suggestion is using the code as source of truth. Another option is to change the default in code to match this line. But I think it makes sense to have all four assertions checked by default, personally.